src/rpc/service_eth.go

```diff
@@ -99,11 +99,10 @@ func (s *EthService) NewTimeEvent(ctx context.Context, newTime int64) {
 
 				utils.Logger.Info().Msg("Transaction sent internally: " + txHash.Hex())
 
-				txInfo := cache.TransactionInfo{info.Tx, newTime}
+				txInfo := cache.TransactionInfo{Tx: info.Tx, CachedTime: newTime}
```

Comment:
Can you please squash this into the previous commit?

Reply:
I opened a new PR and will be closing this one: https://github.com/shutter-network/encrypting-rpc-server/pull/32